Slave.h

```diff
@@ -50,6 +50,7 @@ class Slave
     typedef std::set<std::pair<std::string, std::string>> table_order_t;
     typedef std::map<std::pair<std::string, std::string>, callback> callbacks_t;
     typedef std::map<std::pair<std::string, std::string>, filter> filters_t;
+    typedef std::map<std::pair<std::string, std::string>, struct_callback> struct_callbacks_t;
```

Comment:
If I understand properly the meaning of the word _struct_ here, then it's possible better to replace it with a name _DDL_: `DDL_callback` or `DDL_change_callback`.

Reply:
The original name was picked based on the idea that it should be related to Slave::createDatabaseStructure.